libethereum/Executive.cpp

```diff
@@ -283,64 +287,48 @@ bool Executive::call(CallParameters const& _p, u256 const& _gasPrice, Address co
 		m_gas = _p.gas;
 		if (m_s.addressHasCode(_p.codeAddress))
```

Comment:
I'm not sure if that changes anything, but this function will now return true during the creation phase of `_p.codeAddress`, while it returned false previously.